RELEASES.md

```diff
@@ -6,37 +6,49 @@ Our versioning scheme for LME adheres to [SEMVER 2.0](https://semver.org/):  X.Y
 The patch versions will generally adhere to the following guidelines:
 1. Major SEMVER: Denotes a major release, e.g., a new capability, or LME architecture change.
 2. Minor SEMVER: Denotes updates which are less than major but introduces noticeable changes.
-3. Patch SEMVER: Fix bug issues or vulnerability issues but do not introduce new features or updates.
+3. Patch SEMVER: Fix product breaking bugs, or vulnerabilities, or key documentation issues, but do not introduce new features or updates.
 
 ### Timelines
 
 Development lifecycle timelines will vary depending on project goals, tasking, community contributions, and vision.
 
 
-## Current Release Branch:
-To determine the current release branch, it will either be clearly documented in our wiki or on our public [project](https://github.com/orgs/cisagov/projects/68) board. The below example can also be used to determine our current release branch.
+## Branch Convention:
+We are using a Github flow denoted by: 
+![git-flow](/docs/imgs/git-flow.png)
 
-- For example, if the current latest release (as seen on the main [README](/README.md)) version `1.1.0`, and the `release-*` branches are: `release-1.1.1` and `release-1.2.0` then the `1.2.0` branch would be the branch where submit the PR, since it is the closest release that is a Major or Minor release, while 1.1.1 is a patch release.
+The team requests a brief description if you submit a fix for a current issue on the public project, that context will allow us to help determine if it warrants inclusion. If the PR is well documented following our processes in our CONTRIBUTING.md, it will most likely be worked into LME. We value inclusion and recognize the importance of the open-source community.
```

Comment:
```suggestion
The team requests a brief description if you submit a fix for a current issue on the public project, that context will allow us to help determine if it warrants inclusion. If the PR is well documented following our processes in our [CONTRIBUTING.md](https://github.com/cisagov/LME/blob/main/CONTRIBUTING.md), it will most likely be worked into LME. We value inclusion and recognize the importance of the open-source community.
```